util/credits.txt

```diff
@@ -12,6 +12,7 @@ Alexandre Alapetite
 AlfredSK
 Andi Stadler
 Andreas H.
+Andreas Neustifter
```

Comment:
You don't need to update the credits.txt yourself. It is created prior to a release with a script from the various sources one can contribute to Friendica ;-)

Reply:
Should I revert that change or is it just overwritten with the next release?

Reply:
You can leave it, you just didn't have to do it 😉